tests/scm/SCMPrimitiveUTest.cxxtest

```diff
@@ -200,4 +225,36 @@ void SCMPrimitiveUTest::test_throw(void)
 	logger().debug("END TEST: %s", __FUNCTION__);
 }
 
+void SCMPrimitiveUTest::test_parallel(void)
+{
+	logger().debug("BEGIN TEST: %s", __FUNCTION__);
+
+	as->clear();
+
+	// Create "mango" for creating, and "tango" for deleting
+	MyExampleClass *mtc = new MyExampleClass(as, 42);
+	define_scheme_primitive("mango", &MyExampleClass::my_more_func_1, mtc);
+	define_scheme_primitive("tango", &MyExampleClass::my_more_func_2, mtc);
+
+	eval->eval("(use-modules (ice-9 threads))");
+	eval->eval_h(R"(
```

Comment:
C++ string literal.  Just make it easier to type in the scheme code without worrying about the the quote.